docs/quickstarts/nextjs.mdx

```diff
@@ -118,55 +125,88 @@ The [`<ClerkProvider>`](/docs/components/clerk-provider) component provides acti
-export default authMiddleware({});
+    export const config = {
+      // Protects all routes, including api/trpc.
+      // Edit this to allow other routes to be public.
+      matcher: ["/((?!.+\\.[\\w]+$|_next).*)", "/", "/(api|trpc)(.*)"],
+    };
+    ```
 
-export const config = {
-  matcher: ["/((?!.+\\.[\\w]+$|_next).*)", "/", "/(api|trpc)(.*)"],
-};
-```
+### Try accessing your app
+
+Access your app to verify that authentication is enabled:
 
-**Try accessing your app now by visiting [`http://localhost:3000`](http://localhost:3000).** The Middleware will redirect you to your Sign Up page, provided by Clerk's [Account Portal](/docs/account-portal/overview) feature. If you want to make other routes public, check out the [`authMiddleware` reference page](/docs/references/nextjs/auth-middleware). Go ahead and sign up to get access to your application. Your app is now fully protected by Clerk!
+1. Run your project with the following terminal command from the root directory of your project:
+    <CodeBlockTabs type="installer" options={["npm", "yarn", "pnpm"]}>
+      ```bash filename="terminal"
+      npm run dev
+      ```
+
+      ```bash filename="terminal"
+      yarn dev
+      ```
+
+      ```bash filename="terminal"
+      pnpm dev
+      ```
+
+    </CodeBlockTabs>
+2. Visit [`http://localhost:3000`](http://localhost:3000) to access your app. The Middleware will redirect you to your Sign Up page, provided by Clerk's [Account Portal](/docs/account-portal/overview) feature.
+3. Sign up to gain access to your application.
+4. Visit your app again. You should be able to access it now.
 
 ### Embed the `<UserButton />`
 
-Clerk offers a set of [prebuilt components](/docs/components/overview) to add functionality to your app with minimal effort. The [`<UserButton />`](/docs/components/user/user-button) allows users to manage their account information and log out, completing the full authentication circle.
+[`<UserButton />`](/docs/components/user/user-button) is a [prebuilt component](/docs/components/overview) that allows users to:
+  - Log in.
+  - Manage their account information.
+  - Log out.
 
-Add the `<UserButton />` anywhere inside `<ClerkProvider>` in your app. First, add `import { UserButton } from "@clerk/nextjs";` to the top of your file. Then, add `<UserButton afterSignOutUrl="/"/>`. The `afterSignOutUrl` prop lets you customize what page the user will be redirected to after sign out.
+It must be a child of [`<ClerkProvider>`](/docs/components/clerk-provider).
+
+Use the following code to add a `<UserButton />` to your app: 
 
 <CodeBlockTabs type="router" options={["App Router", "Pages Router"]}>
-```tsx filename="app/page.tsx" {1,6}
+```tsx filename="app/page.tsx" {1,8}
 import { UserButton } from "@clerk/nextjs";
 
 export default function Home() {
   return (
     <div className="h-screen">
+      // Use afterSignOutUrl to redirect
+      // users after signing out

```

Comment:
This prop will be unnecessary in v5 which is slated to go out this/next week

Reply:
also, these are the incorrect comments for JSX. if we want to keep these comments for the v4 docs, the proper comments use {/* */} formatting.

Reply:
Great call, I missed that one 🤦‍♂️